10.md

```diff
@@ -1,15 +1,22 @@
 NIP-10
 ======
 
-
-On "e" and "p" tags in Text Events (kind 1).
---------------------------------------------
+Text Notes and Threads
+----------------------
 
 `draft` `optional`
 
+This NIP defines `kind:1` as a simple plaintext note.
+
 ## Abstract
 This NIP describes how to use "e" and "p" tags in text events, especially those that are replies to other text events.  It helps clients thread the replies into a tree rooted at the original event.
 
+The `.content` property contains some human-readable text. 
+
+`e` and `p` tags can be used to define note threads, replies and mentions. 
+
+Markup languages such as markdown and HTML SHOULD NOT be used. 
```

Comment:
I feel like this could be cleaned up. It sort of jumps back and forth between the kind 1 basic definition and threading details.